packages/worker/src/api/controllers/global/auth.ts

```diff
@@ -197,7 +170,9 @@ export const googlePreAuth = async (ctx: any, next: any) => {
   const strategy = await google.strategyFactory(config, callbackUrl, users.save)
 
   return passport.authenticate(strategy, {
-    scope: ["profile", "email"],
+    scope: ["profile", "email", "https://www.googleapis.com/auth/spreadsheets"],
```

Comment:
I think we can remove the spreadsheets addition on this one, it's added to it's own passport integration in `/middleware/passport/datasource/google.js`